ssz-rs/Cargo.toml

```
@@ -16,18 +16,24 @@ exclude = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["serde"]
+default = ["serde", "std"]
+std = [
+    "bitvec/std",
+    "sha2/std",
+    "num-bigint/std",
+    "serde/std"
+]
 serde = ["dep:serde", "dep:hex"]
 
 [dependencies]
 thiserror = "1.0.25"
-bitvec = "1.0.0"
+bitvec = { version = "1.0.0", default-features = false, features=["alloc"]}
```

if we go w/ the `default` features, then we can do this:
```suggestion
bitvec = { version = "1.0.0", default-features = false }
```

---

This doesn't work because if feature `std` isn't enabled, we still need the `alloc` feature of `bitvec`.